x-pack/plugin/rollup/src/main/java/org/elasticsearch/xpack/rollup/Rollup.java

```diff
@@ -85,6 +85,8 @@
     public static final String ROLLUP_TEMPLATE_VERSION_FIELD = "rollup-version";
     public static final String ROLLUP_TEMPLATE_VERSION_PATTERN =
             Pattern.quote("${rollup.dynamic_template.version}");
+    public static final String ROLLUP_NULL_VALUE_PLACEHOLDER = "ROLLUP_NULL_VALUE_PLACEHOLDER";
+    public static final String ROLLUP_NULL_VALUE = "__ROLLUP_NULL_VALUE_PLACEHOLDER__";
```

Comment:
These look like they might be for the same purpose, maybe we should add comments to them to explain why they are both needed and what for?